module/Olcs/src/Controller/DashboardController.php

```diff
@@ -76,6 +76,12 @@ public function topsreportAction()
                 'refreshToken' => $session['RefreshToken']]
         );
         $response = $this->handleCommand($redirectCmd);
+
+        if (!$response->isOk()) {
+            $this->flashMessenger()->addErrorMessage($response->getResult()['messages'][0]);
```

Comment:
Will `$response->getResult()['messages'][0]` always be available in the response?

Reply:
Yes, if the backend throws the exception this is here to catch there will be 1 message set in that array. But i suppose the isOk() could fail for another reason (backend is very broken in other ways, or unavailable)